src/server/models/Agency.ts

```diff
@@ -0,0 +1,228 @@
+import mongoose from 'mongoose';
+import { IAgency } from '@/types/IAgency';
+
+enum CurrentStatus {
```

Comment:
this should be moved to uw-211/src/utils/types/constants/... so that it can be more easily changed and access elsewhere if needed. We could probably name it something like AgencyStatus